Automattic/wp-calypso
# Remove reader/post-byline

Removes the redundant `reader/post-byline` component. Since the Reader Refresh launch, the byline has been generated by `blocks/reader-post-card/byline.jsx` and `blocks/reader-full-post/header.jsx` instead.

### To test

Visit the Reader (e.g. http://calypso.localhost:3000/read/feeds/15197253) and ensure that the byline is displayed correctly on both on post cards and full post view.
